app/_components/TestComponent/BadComponent.tsx

```
@@ -0,0 +1,60 @@
+    // Very bad: Using dangerouslySetInnerHTML
+    const htmlString = `<h1>Dangerous HTML Injection</h1>`;
+
+    return (
+        <div>
+            <h1 style={{ color: "red", fontSize: "20px", fontWeight: 900 }}>
+                This is a bad React component
+            </h1>
+
+            {/* Bad: No accessibility features */}
+            <button
+                onClick={handleClick}
```

ESLint

[error] Callbacks must be listed after all other props

(react/jsx-sort-props)